src/Microsoft.TemplateEngine.Utils/DefaultTemplateEngineHost.cs

```diff
@@ -64,7 +64,6 @@ public void UpdateLocale(string newLocale)
 
         public virtual void LogMessage(string message)
         {
-            //Console.WriteLine("LogMessage: {0}", message);
             Console.WriteLine(message);
```

Comment:
This still uses `Console.WriteLine`. Is this intended?

Reply:
Yes, it is intended. If host doesn't override it, the output will be done to console.